alembic/ddl/sqlite.py

```diff
@@ -33,17 +33,21 @@ def add_constraint(self, const):
         if const._create_rule is None:
             raise NotImplementedError(
                 "No support for ALTER of constraints in SQLite dialect"
+                "Try to enable batch-mode during auto-migration"
```

Comment:
Let's make it like:

    Please refer to the batch mode feature which allows for SQLite migrations using a copy-and-move strategy.

Reply:
ok, done